.github/workflows/generate-cicd-zip.yaml

```diff
@@ -0,0 +1,36 @@
+name: Generate CICD Zip
```

Comment:
For the issue you mentioned, two options
1. Put both in the same workflow.
2. add the same `concurrency` group to both workflows so that one of them will be queued.